stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -22,6 +25,9 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+{{- if .Values.controller.rbac.enabled }}
+      serviceAccountName: {{ template "fullname" . }}
+{{- end }}
```

Comment:
I believe you'll also want to apply this to the [`controller-daemonset.yaml`](https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/templates/controller-daemonset.yaml#L24).